app/goal_routes.py

```diff
@@ -0,0 +1,74 @@
+from flask import Blueprint, jsonify, abort, make_response, request
```

Comment:
Since there are more than one route file, I'd recommend putting these all in a folder so routes are collocated in the same way models are. Works totally fine for small projects but for larger ones, it would be unwieldy to have all the routes in the application root directory.